contracts/Quest.sol

```diff
@@ -49,7 +49,6 @@ contract Quest is ReentrancyGuardUpgradeable, PausableUpgradeable, OwnableUpgrad
         address protocolFeeRecipient_
     ) external initializer {
         if (endTime_ <= block.timestamp) revert EndTimeInPast();
-        if (startTime_ <= block.timestamp) revert StartTimeInPast();
```

Comment:
I think we're gonna need to remove this test?

Can we also remove the `error StartTimeInPast()` from the interface? Or are we going to need to keep for upgradability?